swagger.yml

```diff
@@ -2154,6 +2154,64 @@ paths:
           schema:
             $ref: "#/definitions/error"
 
+  /repositories/{repository}/metadata/meta_range/{meta_range}:
+    parameters:
+      - in: path
+        name: repository
+        required: true
+        type: string
+      - in: path
+        name: meta_range
+        required: true
+        type: string
+    get:
+      tags:
+        - metadata
+      operationId: getMetaRange
+      summary: return URI to a meta-range file
+      # TODO(ariels): Actually redirect to S3.
+      responses:
+        200:
+          description: meta-range URI
+          headers:
+            Location:
+              type: string
```

Comment:
Why did we prefer to return the response just as a header for this case?
All of our API usually return the response as JSON body.
We can have `location` (bool) parameter to control if we like to get the response just as a location header for a client that doesn't want to parse the response body.

Reply:
This is *almost* a redirect action.  In fact, if we go ahead with signed URLs then it _will_ be a redirect.  I prefer to abuse HTTP semantics here, at least a bit.